interceptor/config/metrics.go

```diff
@@ -7,19 +7,19 @@ import (
 // Metrics is the configuration for configuring metrics in the interceptor.
 type Metrics struct {
 	// Sets whether or not to enable the Prometheus metrics exporter
-	OtelPrometheusExporterEnabled bool `envconfig:"KEDA_HTTP_OTEL_PROM_EXPORTER_ENABLED" default:"true"`
+	OtelPrometheusExporterEnabled bool `envconfig:"OTEL_PROM_EXPORTER_ENABLED" default:"true"`
 	// Sets the port which the Prometheus compatible metrics endpoint should be served on
-	OtelPrometheusExporterPort int `envconfig:"KEDA_HTTP_OTEL_PROM_EXPORTER_PORT" default:"2223"`
+	OtelPrometheusExporterPort int `envconfig:"OTEL_PROM_EXPORTER_PORT" default:"2223"`
 	// Sets whether or not to enable the OTEL metrics exporter
-	OtelHTTPExporterEnabled bool `envconfig:"KEDA_HTTP_OTEL_HTTP_EXPORTER_ENABLED" default:"false"`
+	OtelHTTPExporterEnabled bool `envconfig:"OTEL_EXPORTER_OTLP_METRICS_ENABLED" default:"false"`
 	// Sets the HTTP endpoint where metrics should be sent to
-	OtelHTTPCollectorEndpoint string `envconfig:"KEDA_HTTP_OTEL_HTTP_COLLECTOR_ENDPOINT" default:"localhost:4318"`
+	OtelHTTPCollectorEndpoint string `envconfig:"OTEL_EXPORTER_OTLP_METRICS_ENDPOINT" default:"localhost:4318"`
 	// Sets the OTLP headers required by the otel exporter
-	OtelHTTPHeaders string `envconfig:"KEDA_HTTP_OTEL_HTTP_HEADERS" default:""`
+	OtelHTTPHeaders string `envconfig:"OTEL_EXPORTER_OTLP_METRICS_HEADERS" default:""`
 	// Set the connection to the otel HTTP collector endpoint to use HTTP rather than HTTPS
-	OtelHTTPCollectorInsecure bool `envconfig:"KEDA_HTTP_OTEL_HTTP_COLLECTOR_INSECURE" default:"false"`
+	OtelHTTPCollectorInsecure bool `envconfig:"OTEL_EXPORTER_OTLP_METRICS_INSECURE" default:"false"`
 	// Set the interval in seconds to export otel metrics to the configured collector endpoint
-	OtelMetricExportInterval int `envconfig:"KEDA_HTTP_OTEL_METRIC_EXPORT_INTERVAL" default:"30"`
+	OtelMetricExportInterval int `envconfig:"OTEL_EXPORTER_OTLP_METRICS_EXPORT_INTERVAL" default:"30"`
```

Comment:
I think that we shouldn't handle these vars in our code and delegate them directly to the SDK. Based on docs, it's already managing them:
- http: https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp
- grpc: https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc

(100% optional for this PR) We probably have to include another extra var: `OTEL_EXPORTER_OTLP_PROTOCOL`, based on it enable the HTTP exporter or the gRPC exporter. WDYT?

Reply:
Ah got you. Yeah that's a fair callout and also means less code should be needed too tbf. Will push an update when I get some time 👍

Reply:
@zorocloud any update on this please?

Reply:
Latest update should delegate the env var management to the OTEL SDK now @JorTurFer @zroubalik.

> 100% optional for this PR) We probably have to include another extra var: OTEL_EXPORTER_OTLP_PROTOCOL, based on it enable the HTTP exporter or the gRPC exporter. WDYT?

Apologies @JorTurFer, thought I responded to this! Agree re adding in an option for the gRPC exporter too. I'd prefer to leave it out of this PR though and can raise it in a separate one.

Reply:
Sorry for the slow response, I've been involved on a huge working peak :( 
yeah, a followup PR is more than enough, thanks for your effort! ❤️